implementations/rust/ockam/ockam_command/tests/bats/nodes_unit.bats

```diff
@@ -31,10 +31,50 @@ teardown() {
   assert_success
 }
 
-@test "node - can recreate a foreground node after it was stopped" {
+@test "node - can recreate a background node after it was killed" {
+  # This test emulates the situation where a node is killed by the OS
+  # on a restart or a shutdown. The node should be able to restart without errors.
+  n="$(random_str)"
+  run "$OCKAM" node create "$n"
+  assert_success
+
+  # Fail to create a node with the same name
+  run "$OCKAM" node create "$n"
+  assert_failure
+
+  kill -9 $(cat $OCKAM_HOME/nodes/$n/pid)
+  sleep 0.2
```

Comment:
200ms may be a bit agressive

Reply:
you could try something like this:
```while kill -9 $(cat $OCKAM_HOME/nodes/$n/pid) 2>/dev/null; do sleep 0.1; done;```

kill will fail if the process doesn't exist anymore and will quit the loop

Reply:
I like that, thanks!

Reply:
done